### ISSUE TITLE
Disable ssr

### ISSUE THREAD

**Issue by KirillRomanuk** [Akryum/vue-cli-plugin-ssr#42]
Is there any way to disable SSR after installing plugin? Maybe something like <no-ssr> tag in nuxt.js or any way to disable it globally for full project?

**Comment by chanlito** [comment]
Just curious you use SSR plugin but want to disable it?

**Comment by fenixphp** [comment]
How to disable ssr for certain pages?

**Comment by PureDizzi** [comment]
npm i vue-no-ssr

> Just curious you use SSR plugin but want to disable it?

Some plugins might not work in a server environment.

**Comment by cainrus** [comment]
I want to generate multiple bundles with ssr & wihtout ssr.
I want to disable ssr on highload.
is it possible?

**Comment by vroad** [comment]
Doesn't this plugin support something similar to nuxt's `{ssr: false}`?

https://nuxtjs.org/guide/plugins#client-side-only

**Comment by oboshto** [comment]
Also interested in `no-ssr` prop or similar

**Comment by franciscolourenco** [comment]
The plugin preserves `yarn serve` and `yarn build`. Would indeed be nice to keep them working in parallel.

> Just curious you use SSR plugin but want to disable it?

@chanlito The static version this is useful to take advantage of services like Now, Netlify, Render, etc, to automatically create a deployment for each pull request.

**Comment by bpolaszek** [comment]
Hello there,

Any news on this? I tried using the `skipRequests` option in `vue.config.js` but then all matched routes in this method result in a 404 instead of falling back to the standard SPA.

**Comment by cainrus** [comment]
Sadly, but any vue ssr plugin is poorly supported right now. You can try to use https://universal-vue.github.io/

**Comment by bpolaszek** [comment]
Yes, I'm currently giving it a try and it seems to fit so far :-) Thanks!

PS: By the way the correct link https://universal-vue.github.io/docs/ 🙂
